automation_tools/scripts/ga-migration/README.md

```diff
@@ -1,47 +1,74 @@
 # ga-migration
 
+This folder contains the scripts used to migrate a repository from Travis CI to GitHub actions.
+
+The migration pipeline is *idempotent*, meaning you should be able to run it how many times you want, getting the same results.
+
+The `utils.py` file contains the tasks implementations, while `main.py` imports and applies them according to our needs. `gitflow.py` handles the higher level git and github procedures.
+
 ## Install
 
-```console
-$ virtualenv ~/.virtualenvs/ga-migration
-$ source ~/.virtualenvs/ga-migration/bin/activate
-$ pip install -r requirements.txt
+```bash
+# Activate a virtualenv and install pip requirements
+virtualenv ~/.virtualenvs/ga-migration
+source ~/.virtualenvs/ga-migration/bin/activate
+pip install -r requirements.txt
 ```
 
 ## Migration
 
-Apply migration patches on a cloned repository:
+To apply migration patches on a cloned repository, run `python main.py --targetpath=invenio-charts-js`.
```

Comment:
Minor: Probably `invenio-charts-js` is not the best example to put as it's a JS package and where generating CI tests for python packages 😅

Reply:
Do you have any idea for "simpler" repositories?

Reply:
Any Python package, e.g. `invenio-records`.